devel/mps_load_test.sh

```diff
@@ -208,7 +202,8 @@ cd "$src/devel"
 
 # Build a version of mod_pagespeed with all optimizations enabled, but with
 # a build that includes DCHECKs.
-make -j8 CONF=$compile_mode apache_trace_stress_test_server \
```

Comment:
@morlovich done - see e5b14ad6b4efbd8f668692cc8d5eeb24fa0741c9.
So we only have `GIT_BIN` now.
This seems to work fine for me, do you foresee any issues with it?